src/builder/create_embed.rs

```diff
@@ -100,7 +100,10 @@ impl CreateEmbed {
         self
     }
 
-    /// Set the image associated with the embed. This only supports HTTP(S).
+    /// Set the image associated with the embed.
+    ///
+    /// Refer [Discord Documentation](https://discord.com/developers/docs/reference#uploading-files)
+    /// for allowed characters in `url`.
```

Comment:
The url isn't the problem, is it not? Its Discord filtering the characters allowed in filenames on their cdn.

Reply:
Yeah, I mentioned `url` because I thought it would make sense to the user when calling the function. Changed it to just rules for naming local attachments as you said.